lahja/tools/benchmark/process.py

```diff
@@ -42,15 +44,26 @@ def start(self) -> None:
 
     def stop(self) -> None:
```

Comment:
This is great! multiprocessing should have this built in rather than defaulting to `SIGTERM` :cry: